test/tests/functional/pbs_release_limited_res_suspend.py

```diff
@@ -848,3 +848,84 @@ def test_normal_user_unable_to_see_res_released(self):
         self.assertFalse("resource_released_list.mem" in attrs[0],
                          "Normal user can see resources_released_list "
                          "which is not expected")
+
+    def test_if_node_gets_oversubscribed(self):
+        """
+        Check if the node gets oversubscribed if a filler job runs
+        on resources left on the node after suspension.
+        """
+        # Set mem in restrict_res_to_release_on_suspend server attribute
+        a = {ATTR_restrict_res_to_release_on_suspend: 'mem'}
+        self.server.manager(MGR_CMD_SET, SERVER, a)
+
+        a = {ATTR_sched_preempt_enforce_resumption: True}
+        self.server.manager(MGR_CMD_SET, SCHED, a)
+
+        j1 = Job(TEST_USER)
+        j1.set_attributes({ATTR_l + '.select': '1:ncpus=2:mem=512mb'})
+        jid1 = self.server.submit(j1)
+        self.server.expect(JOB, {ATTR_state: 'R'}, id=jid1)
+
+        # Submit a filler job
+        j2 = Job(TEST_USER)
+        j2.set_attributes({ATTR_l + '.select': '1:ncpus=3',
```

Comment:
You never set the number of ncpus on the machine.  This job will not run on a node with 1 cpu.

Reply:
setup in this test sets the node ncpus to 4